# Generated for NaoQi method ALFaceTracker.isActive
# Return true if the face Tracker is running.

---
# true if the face Tracker is running.
bool value


